cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -42,36 +41,33 @@ def _form_data_cluster_mapping(second_provider, provider):
 
 
 def _form_data_datastore_mapping(second_provider, provider, source_type, target_type):
-    source_datastores_list = second_provider.data.get('datastores')
-    target_datastores_list = provider.data.get('datastores')
-
-    if not source_datastores_list or not target_datastores_list:
-        pytest.skip("No data for source or target cluster in providers.")
-
+    source_datastores_list = second_provider.data.get('datastores', [])
+    target_datastores_list = provider.data.get('datastores', [])
     # assuming, we just have 1 datastore of each type
-    source_datastore = [d.name for d in source_datastores_list if d.type == source_type][0]
-    target_datastore = [d.name for d in target_datastores_list if d.type == target_type][0]
+    source_datastore = [d.name for d in source_datastores_list if d.type == source_type]
+    target_datastore = [d.name for d in target_datastores_list if d.type == target_type]
+    if (not source_datastores_list or not target_datastores_list or
+            not source_datastore or not target_datastore):
+        pytest.skip("No data for source or target datastore in providers.")
```

Comment:
Thanks for descriptive error messages :D

Reply:
:see_no_evil: